Formula/lib/libmsquic.rb

```diff
@@ -6,6 +6,10 @@ class Libmsquic < Formula
       revision: "4922536a9b0b5d1d5c14f907eb3bd0977201123d"
   license "MIT"
 
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
   bottle do
```

Comment:
```suggestion
  end

  bottle do
```

Reply:
Haha, thanks. Blame `brew style --fix` for that one, it put it there 😆